Refactor updating/installing/running Zwift into their own scripts.

hobeone opened the pull request:
First pass at addressing #96

netbrain commented on zwift_common.sh:
this seems a bit overengineered when it only holds a single environment var and sourced into two other files.

netbrain replied:
These script files also need to be added to the zwift_push yaml for github actions.

netbrain reviewed (commented):
Mostly looks good, need to be added to the github action file for script changes tho.

hobeone commented:
Added to the github action file. 

Also fixed an issue in entrypoint where we need to find the home directory of "user" and can't just use $HOME because it's being run as root.

I've run the docker image in both normal and "update" mode and it seems to be doing the right thing.

hobeone commented:
Hmm.  Is there a skew between the Dockerfile that the workflow uses and head?  Does the new Dockerfile need to be pushed somehow so it get's pulled when the workflow does `docker pull netbrain/zwift:latest` ?
